frontend/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Usually front and back ends live in their own repos. Could we separate out this code into another repo where everything is just js related?

Reply:
sure thing